README.md

```diff
@@ -127,6 +127,10 @@ revese_proxy = [
 ]
 ```
 
+To add a custom certificate, you must use a non-slim non-webpki image. Then mount `/usr/local/share/ca-certificates` in the container with your desired CAs each in a file like `myca.crt`. The certificates are accepted in PEM format but extension must be `crt`.
+
+e.g. `-v rpxy/ca-certificates:/usr/local/share/ca-certificates`
+
```

Comment:
Yes, this addition is useful for users. But it is `docker`-specific comments. So I think this should be moved to `./docker/README.md`.

Reply:
Thanks for the update, and also thanks for the information. I think we should merge this change for now, and the thing related to alpine should be treated later. So I made another issue #188 for the alpine-based docker image.